Common/include/option_structure.hpp

```diff
@@ -826,7 +826,8 @@ enum ENUM_UPWIND {
   LAX_FRIEDRICH = 15,         /*!< \brief Lax-Friedrich numerical method. */
   AUSMPLUSUP = 16,            /*!< \brief AUSM+ -up numerical method (All Speed) */
   AUSMPLUSUP2 = 17,            /*!< \brief AUSM+ -up2 numerical method (All Speed) */
-  AUSMPWPLUS = 18            /*!< \brief AUSMplus numerical method. (MAYBE for TNE2 ONLY)*/
+  AUSMPWPLUS = 18,            /*!< \brief AUSMplus numerical method. (MAYBE for TNE2 ONLY)*/
+  SCALAR_ADVECTION = 19
```

Comment:
@EvertBunschoten Good to make an option for this. Now that we're touching ENUM_UPWIND anyway, can you change it to an enum class? In that way we can get rid of explicitly numbering the options.